forge-gui/res/cardsfolder/a/ambergris_citadel_agent.txt

```diff
@@ -4,10 +4,10 @@ Types:Legendary Creature Dwarf Cleric
 PT:3/2
 K:Haste
 K:Specialize:3
-T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, you may discard your hand and draw two cards.
-SVar:TrigDiscard:DB$ Discard | Optional$ True | Mode$ Hand | Defined$ You | SubAbility$ DBDraw
-SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 2
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDiscardDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, you may discard your hand and draw two cards.
```

Comment:
Not everything is a cost, these are the rules:
> 118.12. Some spells, activated abilities, and triggered abilities read, "[Do something]. If [a player] [does, doesn't, or can't], [effect]." Or "[A player] may [do something]. If [that player] [does, doesn't, or can't], [effect]." The action [do something] is a cost, paid when the spell or ability resolves.
> 118.12a. Some spells, activated abilities, and triggered abilities read, "[Do something] unless [a player does something else]." This means the same thing as "[A player may do something else]. If [that player doesn't], [do something]."

Reply:
Ah thank you, I should have checked 🤦 

Back to the drawing board